packages/trip-form/src/components.story.tsx

```diff
@@ -130,7 +130,7 @@ sliderSelector.parameters = {
 export const dateTimeSelector = makeStory(Core.DateTimeSelector, {
   date: "2020-02-15",
   dateFormatLegacy: "YY-M-d",
-  departArrive: "NOW",
+  departArrive: "DEPART",
```

Comment:
No they don't, this is just for the story because when it's set to NOW you don't get to see the actual date time selector. Ideally this would be a control in the story but the way we've set up the story here makes that difficult, should probably redo that at some point.

Reply:
Ah I see. Ok looks good!